src/TableColumn/ActionButtons.php

```diff
@@ -111,7 +111,7 @@ public function addModal($button, $title, $callback, $owner = null, $args = [])
     {
         $owner = $owner ?: $this->owner->owner;
 
-        $modal = \atk4\ui\Modal::addTo($owner, [compact('title'), 'appStickyCb' => true]);
+        $modal = \atk4\ui\Modal::addTo($owner, [is_array($title) ? $title : compact('title'), 'appStickyCb' => true]);
```

Comment:
Replace all `compact('xx', ...)` usages to `['xx' => $xx, ...]`, compact is not refactorable.

Reply:
Corrected